docs/sql-ref-syntax-ddl-create-table-hiveformat.md

```diff
@@ -203,6 +218,17 @@ CREATE EXTERNAL TABLE family (id INT, name STRING)
     STORED AS INPUTFORMAT 'com.ly.spark.example.serde.io.SerDeExampleInputFormat'
         OUTPUTFORMAT 'com.ly.spark.example.serde.io.SerDeExampleOutputFormat'
     LOCATION '/tmp/family/';
+
+--Use `CLUSTERED BY` clause to create bucket table without `SORTED BY`
+CREATE TABLE TEST1(ID INT, AGE STRING)
```

Comment:
nit: To follow the format of the other examples, `CREATE TABLE clustered_by_test1 (ID ...`?